samples/graph-chat-migration/csharp/GraphChatMigration/GraphClient/SimpleGraphClient.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Microsoft.Graph.Beta;
```

Comment:
Please add Copyright

// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.